packages/protocol/scripts/sourcify-publish.ts

```diff
@@ -33,42 +33,26 @@ import Web3 from 'web3'
  *
  */
 
-interface BuildOptions {
-  network: string
-  buildArtifactsPath: string
-  proposalPath: string
+const _buildTargets = {
+  network: undefined,
+  build_artifacts_path: undefined,
+  proposal_path: undefined,
 }
 
-async function main(buildTargets: BuildOptions) {
-  const artifactBasePath = buildTargets.buildArtifactsPath || './build/contracts'
+async function main(buildTargets: typeof _buildTargets) {
+  const artifactBasePath = buildTargets.build_artifacts_path || './build/contracts'
   const artifactPaths = fs.readdirSync(artifactBasePath)
-  const reportPath = buildTargets.proposalPath || './proposal.json'
+  const reportPath = buildTargets.proposal_path || './proposal.json'
   const report = require(path.join(process.cwd(), reportPath))
   const network = buildTargets.network
   const web3 = new Web3('http://localhost:8545')
   const chainId = await web3.eth.getChainId()
 
   console.log('Uploading sources & metadata')
   console.log('============================')
-  console.log(artifactPaths)
 
   const artifacts = artifactPaths.map((a) => require(path.join(process.cwd(), artifactBasePath, a)))
 
-  if (
-    !isEqual(
-      artifacts.map((a) => a.contractName),
-      report.map((a) => a.contract)
-    )
-  ) {
-    if (report.length !== artifactPaths.length) {
-      throw new Error(
-        `There are ${report.length} addresses to publish, but there are ${artifactPaths.length} artifacts.`
-      )
-    } else {
-      throw new Error(`Contracts names are not equal`)
-    }
-  }
-
   for (const r of report) {
```

Comment:
Not from this PR, but can we use a more descriptive name for the loop variable? E.g. `proposalTransaction` or `proposalItem`.